src/test/java/net/revelc/code/impsort/ImpSortTest.java

```diff
@@ -70,4 +71,12 @@ public void parseGroups() {
         Grouper.parse(" b , * , ab ,com., java , a"));
   }
 
+  @Test
+  public void testIso8859ForIssue3() throws IOException {
+    Path p =
+        Paths.get(System.getProperty("user.dir"), "src", "test", "resources", "Iso8859File.java");
+    Result result =
+        new ImpSort(StandardCharsets.ISO_8859_1.name(), eclipseDefaults, true).parseFile(p);
+    assertTrue(result.getImports().isEmpty());
```

Comment:
could write the result out and ensure it check that it has the proper encoding

Reply:
done